src/routes/OrganizationsList/Organization/Tabs/Settings/Settings.tsx

```diff
@@ -16,6 +16,8 @@ import {
 import {useLocation} from 'react-router-dom';
 import {useCurrentUser} from 'src/hooks/UseCurrentUser';
 import {useOrganization} from 'src/hooks/UseOrganization';
+import {BillingInformation} from './BillingInformation';
+import {CliConfiguration} from './CLIConfiguration';
 
 const GeneralSettings = () => {
```

Comment:
I see this variable defined in [GeneralSettings.tsx](https://github.com/quay/quay-ui/pull/171/files#diff-132113857fa8cffa70ca035d57148eeef72301ecda9ebdf2758f9d8b3c220194R26) and one of it doesn't seem to be referenced anywhere. Could you take a look?